docs-src/clusters/how-to-integrate-elasticsearch-into-a-temporal-cluster.md

```diff
@@ -31,7 +31,7 @@ If you operate a Temporal Cluster using our [Helm charts](https://github.com/tem
 **Persistence configuration**
 
 1. Add the `advancedVisibilityStore: es-visibility` key-value pair to the `persistence` section.
-   The [development_es.yaml](https://github.com/temporalio/temporal/blob/master/config/development_es.yaml) file in the `temporalio/temporal` repo is a working example.
+   [There are several development_es.yaml files](https://github.com/temporalio/temporal/tree/master/config) in the `temporalio/temporal` repo you can look at for example usage.
```

Comment:
```suggestion
   For example usage, you can look at [several development_es.yaml files](https://github.com/temporalio/temporal/tree/master/config) in the `temporalio/temporal` repo.
```
These edits place the goal before the action per https://developers.google.com/style/sentence-structure. The changes also conveniently avoid the construction "There are" (which is what initially caught my eye).